__tests__/ord.test.js

```diff
@@ -14,6 +14,27 @@ describe("Tests for default ORD document", () => {
     expect(document).toMatchSnapshot();
   });
 
+  describe("apiResources", () => {
+    const PACKAGE_ID_REGEX = /^([a-z0-9]+(?:[.][a-z0-9]+)*):(package):([a-zA-Z0-9._\-]+):(v0|v[1-9][0-9]*)$/
+
+    let document;
+
+    beforeAll(()=> {
+      document = ord(csn);
+    })
+
+    test("partOfPackage values are valid ORD IDs ", () => {
+      for (const apiResource of document.apiResources) {
+        expect(apiResource.partOfPackage).toMatch(PACKAGE_ID_REGEX)
+      }
+    })
+
+    test("The partOfPackage references an existing package", () => {
+        for (const apiResource of document.apiResources) {
+          expect(document.packages.find((pck) => pck.ordId == apiResource.partOfPackage)).toBeDefined
+        }
+      })
+  })
```

Comment:
Two checks are added:

1. If the partOfPackage is a valid ORD ID
2. If it references an existing package.ordId
I guess there is no need to check if the packages[].ordId is a valid ORD ID because it is implicitly "covered" by the combination of these two checks.

@ductaily @Fannon @RamIndia : Do you agree or I should add some more tests/checks?